symfony/plugins/orangehrmMaintenancePlugin/modules/maintenance/templates/purgeCandidateSuccess.php

```diff
@@ -0,0 +1,7 @@
+<?php
```

Comment:
Empty file?

Reply:
need to implement candidate functionality. Which is in last of OHRM tickets.

Reply:
remove the candidate purge Action & success files for now since they aren't part of the Employee purge functionality. You can add them later once you start working on the Candidate purge